ghost/i18n/locales/pt-BR/portal.json

```diff
@@ -154,21 +154,21 @@
     "Success! Your account is fully activated, you now have access to all content.": "Sucesso! Sua conta está totalmente ativada, agora você tem acesso a todo o conteúdo.",
     "Success! Your email is updated.": "Sucesso! Seu e-mail foi atualizado.",
     "Successfully unsubscribed": "Inscrição cancelada com sucesso",
-    "Thank you for subscribing. Before you start reading, below are a few other sites you may enjoy.": "Obrigado por se inscrever. Antes de começar a ler, abaixo estão alguns outros sites que você pode gostar.",
-    "Thank you for your support": "Obrigado pelo seu apoio",
-    "Thank you for your support!": "Obrigado pelo seu apoio !",
-    "Thanks for the feedback!": "Obrigado pela avaliação!",
+    "Thank you for subscribing. Before you start reading, below are a few other sites you may enjoy.": "Gratidão por se inscrever. Antes de começar a ler, abaixo estão alguns outros sites que você pode gostar.",
+    "Thank you for your support": "Gratidão pelo seu apoio",
+    "Thank you for your support!": "Gratidão pelo seu apoio !",
```

Comment:
is there an extra space before the !

Reply:
Thank you for the catch ;) I just fixed it